frontend/src/app/projects/project-overview/project-overview.component.html

```diff
@@ -16,10 +16,17 @@
     <mat-icon matSuffix>search</mat-icon>
   </mat-form-field>
 
-  <mat-chip-listbox class="-mt-2" formControlName="projectType">
-    <mat-chip-option value="general">Projects</mat-chip-option>
-    <mat-chip-option value="training">Trainings</mat-chip-option>
-  </mat-chip-listbox>
+  <div class="-mt-2 flex gap-3">
+    <mat-chip-listbox formControlName="projectType">
+      <mat-chip-option value="general">Projects</mat-chip-option>
+      <mat-chip-option value="training">Trainings</mat-chip-option>
+    </mat-chip-listbox>
+
+    <mat-chip-listbox formControlName="projectVisibility">
```

Comment:
```suggestion
    </mat-chip-listbox>

    <div class="border-l"></div>

    <mat-chip-listbox formControlName="projectVisibility">
```

Add a small separator to indicate two different groups.